include/edge_kernels/VOFAdvectionEdgeAlg.h

```diff
@@ -39,7 +39,11 @@ class VOFAdvectionEdgeAlg : public AssembleEdgeSolverAlgorithm
   unsigned dqdx_{stk::mesh::InvalidOrdinal};
   unsigned edgeAreaVec_{stk::mesh::InvalidOrdinal};
   unsigned massFlowRate_{stk::mesh::InvalidOrdinal};
+  unsigned massVofBalancedFlowRate_{stk::mesh::InvalidOrdinal};
   unsigned density_{stk::mesh::InvalidOrdinal};
+
+  double density_liquid_;
```

Comment:
Liquid and gas makes me think this is a 2-phase flow and not arbitrary "multi" phase? Am I thinking about this right?

Reply:
I think liquid and gas are the only phases we will be equipped to deal with in a suite of physics nalu can handle